tests/sentry/uptime/consumers/test_results_consumer.py

```diff
@@ -1046,3 +960,107 @@ def test_check_and_update_regions_removes_disabled(self):
             [("region1", "upsert"), ("region2", "delete")],
             current_minute=5,
         )
+
+
+class ProcessResultSerialTest(ProcessResultTest):
```

Comment:
is the naming of this test wrong?

Reply:
No, it's running in serial mode. I chucked a couple of the batch tests in there because I didn't want to run all the tests a third time